### DISCUSSION TITLE
1.3版本如何查询jvm那些指标数据啊？

### DISCUSSION THREAD

**Discussion by Needy332** [apache/iotdb#13601] [Q&A] [answered]
如 SELECT * FROM root._metric."127.0.0.1:6667"."jvm.threads.peak"  这条语句在0.13版本可以运行在新版本就运行不了了
看了好久文档也没看到相关说明
除了通过prometheus的方式去获取监控数据，还有别的方式吗？

**Comment by 2b3c511** [comment, marked as answer]
0.13版本支持监控数据写回IoTDB，在cli里通过SQL进行查询；1.3版本里不支持，除了通过prometheus查看监控数据，也可以通过JMX来监控服务
